frontend/src/main.js

```diff
@@ -3,15 +3,29 @@ import DashboardPlugin from './plugins/dashboard-plugin'
 import App from './App.vue'
 import i18n from './i18n.js'
 import { loadAllRules } from './validation-rules'
-import ApolloClient from 'apollo-boost'
+import { ApolloClient, ApolloLink, InMemoryCache, HttpLink } from 'apollo-boost'
 import VueApollo from 'vue-apollo'
 import CONFIG from './config'
 
 import { store } from './store/store'
 
 import router from './routes/router'
 
+const httpLink = new HttpLink({ uri: CONFIG.GRAPHQL_URI })
+
+const authLink = new ApolloLink((operation, forward) => {
+  const token = store.state.token
+  operation.setContext({
+    headers: {
+      Authorization: token && token.length > 0 ? `Bearer ${token}` : '',
```

Comment:
Why `token.length > 0` why can this happen? Shouldn't we either have `null` or some token string?

Reply:
Just to make sure that token can not be an empty string (should be valid or null)